packages/core/src/lib/lit-core.ts

```diff
@@ -484,6 +484,17 @@ export class LitCore {
   };
 
   /**
+   * Return the latest blockhash from the nodes
+   * @returns { Promise<string> } latest blockhash
+   */
+  getLatestBlockhash = async (): Promise<string> => {
+    await this.connect();
+
+    return this.latestBlockhash as string;
```

Comment:
If we are going to assert this is a string, we should probably actually check and make sure and throw if for some reason latestBlockhash wasn't set on connect.

Reply:
good catch!